active/website.md

```diff
@@ -0,0 +1,62 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official website, **djangoproject.com** on behalf of the Django Software Foundation
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
+- Members can tag,label and respond to issues.
+- Member can maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas
```

Comment:
I think this is premature. @sabderemane is a board member, she expressed an interest in helping with this too, a good two months before I did. @cgl is a board member who is already involved with the site’s maintenance and might also be keen to be Board Liaison, though she hasn’t said what she’d think of continuing this as a working group. The three of us (and any other interested board member) can figure it out.

```suggestion
- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA
```